install_config/configuring_sdn.adoc

```diff
@@ -271,24 +274,53 @@ If you are already using one SDN plug-in and want to switch to another:
 . Change the `*networkPluginName*` parameter on all
 xref:configuring-the-pod-network-on-masters[masters] and
 xref:configuring-the-pod-network-on-nodes[nodes] in their configuration files.
+
+. Restart the API and master services on all masters.
++
+----
+master-restart api master-restart controllers
+----
+
 ifdef::openshift-origin[]
-. Restart the *origin-master-api* and *origin-master-controller* services on all masters.
 . Stop the *origin-node* service on all masters and nodes.
++
+----
+systemctl stop origin-node
+----
 endif::[]
 ifdef::openshift-enterprise[]
-. Restart the *atomic-openshift-master-api* and *atomic-openshift-master-controllers* services on all masters.
-. Stop the *atomic-origin-node* service on all masters and nodes.
+. Stop the *atomic-openshift-node* service on all masters and nodes.
++
+----
+systemctl stop atomic-openshift-node
+----
 endif::[]
+
 . If you are switching between OpenShift SDN plug-ins, restart the *openvswitch* service on all masters and nodes.
```

Comment:
It's no longer "openvswitch service". We can simply say "restart OpenShift SDN."